packages/manager/apps/web/client/app/private-database/order/sql-database-order.html

```diff
@@ -39,7 +39,8 @@ <h2 data-translate="privateDatabase_order_sql_type_title"></h2>
                                 <span class="oui-radio__label">
                                     <span data-ng-bind-html="'privateDatabase_order_sql_type_' + item.key + '_label' | translate"></span>
                                     <span class="fa fa-question-circle"
-                                          data-uib-tooltip-template="'private-database/order/tooltip/sql-database-order-' + item.key + '.html'"></span>
+                                          data-oui-popover
+                                          data-oui-popover-template="'private-database/order/tooltip/sql-database-order-' + 'item.key' + '.html'"></span>
```

Comment:
A popover ≠ a tooltip. If you turn into a popover maybe a `button` would be more appropriate than a span

Reply:
We could even use `<button type="button" class="oui-popover-button" oui-popover="..."></button>`, it's made for that (and replace the use of FA icons too) :)

ex. v2 https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/oui-angular/popover, or v3 https://ovh-ux.github.io/ovh-ui-kit/?path=/story/directives-popover--help-button

Reply:
Have added `<button>` for popover.